lib/tests/detectors/wpar_detector.cc

```diff
@@ -9,15 +9,42 @@ using namespace whereami::testing::lparstat;
 using namespace std;
 
 SCENARIO("Using the WPAR detector") {
-    WHEN("Running inside a WPAR") {
-        lparstat_fixture lparstat_source {"7.1.0.0", "output/lparstat/wpar.txt"};
+    WHEN("Running on AIX") {
+        WHEN("Running inside an LPAR") {
```

Comment:
Do you think the LPAR WHEN is necessary? i.e. Does it have to run under LPAR for it to run inside WPAR or can it run inside WPAR independently of LPAR? If it's the latter, it might be helpful to remove the WHEN("Running inside an LPAR") part.